docs/sdk/src/reference_docs/frame_currency.rs

```diff
@@ -1,8 +1,55 @@
-//! FRAME Currency Abstractions and Traits
+//! # FRAME Currency Abstractions and Traits
```

Comment:
I think it would be useful to have a table with the comparing the Currency vs Fungible/s terms and a description on how they differ (e.g. locks vs holds, etc). This is usually a place where people get confused and it would be great if we could have a single place to refresh the memory.

Reply:
I agree we can improve this document by adding some more information, maybe we can keep that for another PR, wdyt?